modules/va_forms/app/swagger/va_forms/forms/form_swagger.rb

```diff
@@ -70,42 +74,49 @@ class FormSwagger
             property :form_usage do
               key :description, I18n.t('va_forms.field_descriptions.form_usage')
               key :type, :string
+              key :nullable, true
               key :example, '<p>Use VA Form 10-10EZ if you’re a Veteran and want to apply for VA health care. You must be enrolled in...</p>'
             end
             property :form_tool_intro do
               key :description, I18n.t('va_forms.field_descriptions.form_tool_intro')
               key :type, :string
+              key :nullable, true
               key :example, 'You can apply online instead of filling out and sending us the paper form.'
             end
             property :form_tool_url do
               key :description, I18n.t('va_forms.field_descriptions.form_tool_url')
               key :type, :string
+              key :nullable, true
               key :example, 'https://www.va.gov/health-care/apply/application/introduction'
             end
             property :form_details_url do
               key :description, I18n.t('va_forms.field_descriptions.form_details_url')
               key :type, :string
+              key :nullable, true
               key :example, 'https://www.va.gov/find-forms/about-form-10-10ez'
             end
             property :form_type do
               key :description, I18n.t('va_forms.field_descriptions.form_type')
               key :type, :string
+              key :nullable, true
               key :example, 'benefit'
             end
             property :language do
               key :description, I18n.t('va_forms.field_descriptions.language')
               key :type, :string
               key :example, 'en'
             end
-            property :deleted_at do
+            property :deleted_at, type: %i[string null] do
```

Comment:
I believe the type here should only be `string` which is defined on L111, so I think we can ditch the property setting here.

See: https://swagger.io/specification/#schema-object
> type - Value MUST be a string. Multiple types via an array are not supported.

```suggestion
            property :deleted_at do
```